spec/requests/api/v1/cache_header_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+# Moved multiple request specs here from the controller spec
+
+RSpec.describe 'API cache header verification',
+               type: :request, api: true, version: :v1 do
+
+  let!(:untrusted_application) { FactoryGirl.create :doorkeeper_application }
+  let!(:trusted_application)   { FactoryGirl.create :doorkeeper_application, :trusted }
+  let!(:user_2)       do
+    FactoryGirl.create :user_with_emails,
+                       first_name: 'Bob',
+                       last_name: 'Michaels'
+  end
+
+  let!(:user_2_token) do
+    FactoryGirl.create :doorkeeper_access_token,
+                       application: untrusted_application,
+                       resource_owner_id: user_2.id
+  end
+
+  let!(:untrusted_application_token) do
+    FactoryGirl.create :doorkeeper_access_token,
+                       application: untrusted_application,
+                       resource_owner_id: nil
+  end
+  let!(:trusted_application_token) do
+    FactoryGirl.create :doorkeeper_access_token,
+                       application: trusted_application,
+                       resource_owner_id: nil
+  end
```

Comment:
I know I wrote this, but can you remove the unused `let`s?  I think 3 of them can go.